### DISCUSSION TITLE
Maui Blazor

### DISCUSSION THREAD

**Discussion by Scalee** [LazZiya/XLocalizer#37] [Q&A]
Would it be possible to use XLocalizer with Maui Blazor?

**Comment by LazZiya** [comment]
Not yet! Some dependencies don't have cross platform compatibility. I will check if this can be solved.

Best,
Ziya

**Comment by julianadormon** [comment]
I would love this as well. In fact, I really need it! Any way to sponsor this yet? I was unable to pay via Paypal last time.

**Reply by LazZiya** [reply]
It seems I need to get back and afford more time to update this repo. I was working on another private project for WebAR, and so couldn't find enough time to follow other repos.

Thank you @julianadormon for your willing to support, the sponsorship button should appear in the main repo page :)

**Reply by julianadormon** [reply]
I was able to make a sponsor payment. Thank you for your consideration!

**Reply by LazZiya** [reply]
Thank you @julianadormon for your support, I'm really glad to receive my first ever sponsoring payment :)

**Reply by julianadormon** [reply]
Well, I'm happy to be the first. I hope there is more for you.

**Reply by julianadormon** [reply]
This really makes localization a breeze, I hope you can do something for Maui. So far it has not been much fun doing it the old way.

**Reply by LazZiya** [reply]
I will do my best for Maui, hop it will not take much time.

**Reply by julianadormon** [reply]
That would be awesome. I just had this open in fact: https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/localization?view=net-maui-8.0

**Reply by LazZiya** [reply]
I already checked that link and I still in the early phase of understanding all the localization requirements of MAUI. It seems more complicated and will need a dedicated package of XLocalizer.
